lenovo/legion/t526amr5/README.md

```diff
@@ -0,0 +1 @@
+TODO: add content
```

Comment:
it feels a bit pointless to have this file if we dont actually add anything.
```suggestion
```

Reply:
Ah. When I said "README", I meant adding it to the README.md table in the toplevel of this project.